Entities/Actor.h

```diff
@@ -200,31 +200,24 @@ ClassInfoGetters
     int GetStatus() const { return m_Status; }
 
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Method:          GetHealth
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Gets this Actor's health value.
-// Arguments:       None.
-// Return value:    A const int describing this Actor's health.
-
-    int GetHealth() const { return m_Health; }
-
+	/// <summary>
+	/// Gets this Actor's health value.
+	/// </summary>
+	/// <returns>A const float describing this Actor's health.</returns>
```

Comment:
The float here isn't const. Const here describes the method as a const one, so you should just remove that const from your return comment. And yeah, this was data's screwup, not yours.

Applies below as well.